gef.py

```diff
@@ -3472,18 +3542,15 @@ def db(t, p):
     return db(1, 1)
 
 
-def generate_cyclic_pattern(length):
+def generate_cyclic_pattern(length, cycle=4):
     """Create a `length` byte bytearray of a de Bruijn cyclic pattern."""
     charset = bytearray(b"abcdefghijklmnopqrstuvwxyz")
-    cycle = get_memory_alignment()
-    res = bytearray()
-
+    pattern = bytearray()
     for i, c in enumerate(de_bruijn(charset, cycle)):
```

Comment:
We might be able to clean this up: `return bytearray(itertools.islice(de_bruijn(charset, cycle), length))`